test/scan.js

```diff
@@ -1,21 +1,25 @@
-import 'babel-polyfill';
-import scan from '../src/scan.es6.js';
-import assert from 'assert';
+import "babel-polyfill";
+import scan from "../src/scan.es6.js";
+import assert from "assert";
+import jsdom from "mocha-jsdom";
 
-describe("scan.js", () => {
-  it("is a function", () => {
-    assert(typeof scan === 'function');
+xdescribe("scan.js", () => {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["xdescribe" is not defined.](https://www.codacy.com/app/lagora-franck/proc-edu-ria/file/2546619596/issues/source?bid=3286583&fileBranchId=3287652#l6)